metoncofit/DEA/Lyssiotis_DEA.R

```diff
@@ -0,0 +1,118 @@
+#### Imports ####
```

Comment:
Same here regarding the for loop across each csv file in the directory and the log2fc/zscore